src/web/deployment.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "Web Production Deployment"
```

Comment:
This is very different from the name in the sidenav (and also we use sentence case for titles now). How about we call it "Building for web deployment". If we need a short-title (which is probably good, tab titles being so limited), it could be "Web deployment".

Globally, Web -> web. (https://developers.google.com/style/word-list#letter-W)